torch/_higher_order_ops/torchbind.py

```diff
@@ -69,6 +78,21 @@ def inner(mode, *args, **kwargs):
         )
         out = call_torchbind(*args, **kwargs)
 
+        obj, method, *rest_args = args
+        if isinstance(obj, torch.ScriptObject):
+            ns, class_name = _ns_and_class_name(
+                obj._type().qualified_name()  # type: ignore[attr-defined]
+            )
+            log.warning(
+                "Tracing torchbind method %s.%s with real ScriptObject. This may"
+                " cause the original object being mutated. If this is not intended,"
+                ' You can register a fake class with torch._library.register_fake_class("%s::%s").',
```

Comment:
mm, we should probably error out on this if FakeTensorMode is active

Reply:
We cannot do it in this diff yet because many tests in current export is still tracing with real ScriptObject. We can do this when we turn on fakification by default.

Reply:
ya sounds good